app/assets/javascripts/scheduler.js

```diff
@@ -537,7 +546,7 @@ $(document).ready(function() {
 				course = course[0].match(/([A-Za-z]+)([0-9]+)/);
 				course = new Array(course[1], course[2]);
 			}
-			$('#search-classes').addClass('loading');
+			// $('#search-classes').addClass('loading');
```

Comment:
Tried to leave most it there in case we wanted to re-implement it another way. Can you remove if you think that's best